app.py

```diff
@@ -65,5 +75,76 @@ def delete(self):
             print("Error occured:", str(e.args))
             return Response('{"message":"Server error. Please check logs."}', status=400, mimetype='application/json')
 
+
+@ns_api_v1.route('/customer')
+class Customer(Resource):
+    global customerTable
+   # @marshal_with(store_marshal)
+
+    def get(self):
+        try:
+            cursor = customerTable.find({}, {"password" : 0})   # operation on table to get all data
+            response = []
+            for document in cursor: # iterate through each db result and append to a list
+                response.append(document)
+                return Response('{"response":%s,"message":"Succesfully retreived all documents"}' % dumps(response), status=200, mimetype='application/json')
```

Comment:
This response will output all the data in customer table right which includes hashed passwords as well. I would say you need to marshal this response before sending

https://flask-restplus.readthedocs.io/en/stable/marshalling.html

Reply:
it won't the query i used, will skip the password form the db directly. I mean I can marshall the response but this serves the same purpose.

Reply:
Ok then that's good enough